src/test/java/org/cactoos/experimental/ThreadsTest.java

```diff
@@ -95,13 +96,14 @@ public void failsDueToException() {
             "wraps error into CompletionException",
             () -> new Threads<String>(
                 Executors.newSingleThreadExecutor(),
-                () -> {
+                (Scalar<String>) () -> {
```

Comment:
@victornoel What do you think about introducing extra class as: `ScalarOf` in Cactoos, so that we do not require to to that? 
Maybe just to leave a todo?

Reply:
@fanifieiev done with a todo